### PR TITLE
Sdk code samples

### PR DESCRIPTION
**Pull request by jvc9109**
[FIXED] ~This PR is using an invalid `lang`. When using a valid `lang: PHP` and `label: SDK-3.0` there is a visualization bug. Details in https://github.com/Rebilly/api-definitions/pull/977~

### CODE REVIEW

File: openapi/paths/credentials.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ get:
     - lang: PHP
       source:
         $ref: ../code_samples/PHP/credentials/get.php
+    - lang: PHP-sDK-3.0
```
Comment (2e3s) [inline openapi/paths/credentials.yaml:38]:
```suggestion
    - lang: PHP-SDK-3.0
```

File: openapi/paths/password-tokens@{id}.yaml [hunk 2]
```diff
@@ -27,6 +27,9 @@ get:
     - lang: JavaScript
       source:
         $ref: '../code_samples/JavaScript/password-tokens@{id}/get.js'
+    - lang: PHP-SDK-3.-
```
Comment (2e3s) [inline openapi/paths/password-tokens@{id}.yaml:30]:
```suggestion
    - lang: PHP-SDK-3.0
```

**Comment by 2e3s** [comment]
> When using a valid lang: PHP and label: SDK-3.0 there is a visualization bug. Details in https://github.com/Rebilly/api-definitions/pull/977

Both seem to have the same `lang: PHP-SDK-3.0`?

**Comment by jvc9109** [comment]
@2e3s  I include the change only for `customers.yaml` . I have changed the PR base to make it easier to read.

File: openapi/code_samples/PHP-SDK-3.0/authentication-options/put.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$authenticationOptionsForm = Rebilly\Sdk\Model\AuthenticationOptions::from([]);
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/authentication-options/put.php:3]: `from` method is only useful with non-empty argument or for chained calls.
```suggestion
$authenticationOptionsForm = new Rebilly\Sdk\Model\AuthenticationOptions();
```

File: openapi/code_samples/PHP-SDK-3.0/credentials/post.php [hunk 4]
```diff
@@ -0,0 +1,12 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$customerCredentialForm = Rebilly\Sdk\Model\CustomerCredential::from([]);
+$customerCredentialForm->setCustomerId('customerId');
+$customerCredentialForm->setUsername('test');
+$customerCredentialForm->setPassword('1234');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/credentials/post.php:6]:
```suggestion
$customerCredentialForm = Rebilly\Sdk\Model\CustomerCredential::from([])
    ->setCustomerId('customerId')
    ->setUsername('test')
    ->setPassword('1234');
```

File: openapi/code_samples/PHP-SDK-3.0/credentials@{id}/put.php [hunk 5]
```diff
@@ -0,0 +1,12 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$customerCredentialForm =Rebilly\Sdk\Model\CustomerCredential::from([]);
+$customerCredentialForm->setCustomerId('customerId');
+$customerCredentialForm->setUsername('test');
+$customerCredentialForm->setPassword('1234');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/credentials@{id}/put.php:6]:
```suggestion
$customerCredentialForm = Rebilly\Sdk\Model\CustomerCredential::from([])
    ->setCustomerId('customerId')
    ->setUsername('test')
    ->setPassword('1234');
```

File: openapi/code_samples/PHP-SDK-3.0/customers/post.php [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        Rebilly\Sdk\Model\ContactObject::from([])
+            ->setFirstName('John')
+            ->setLastName('Doe')
+            ->setAddress('Test street 5')
+            ->setEmails([
+                Rebilly\Sdk\Model\ContactEmails::from([])
+                    ->setLabel('main')
+                    ->setValue('johndoe@email.com')
+                    ->setPrimary(true)
+            ])
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/customers/post.php:15]:
```suggestion
                    ->setPrimary(true),
            ]),
```

File: openapi/code_samples/PHP-SDK-3.0/customers@{id}/put.php [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+        Rebilly\Sdk\Model\ContactObject::from([])
+            ->setFirstName('John')
+            ->setLastName('Doe')
+            ->setAddress('Test street 5')
+            ->setEmails([
+                Rebilly\Sdk\Model\ContactEmails::from([])
+                    ->setLabel('main')
+                    ->setValue('johndoe@email.com')
+                    ->setPrimary(true)
+            ])
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/customers@{id}/put.php:15]:
```suggestion
                    ->setPrimary(true),
            ]),
```

File: openapi/code_samples/PHP-SDK-3.0/customers@{id}@lead-source/put.php [hunk 8]
```diff
@@ -0,0 +1,11 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$leadSourceForm = Rebilly\Sdk\Model\LeadSource::from([]);
+$leadSourceForm->setSource('TestSource');
+$leadSourceForm->setCampaign('TestCampaign');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/customers@{id}@lead-source/put.php:5]:
```suggestion
$leadSourceForm = Rebilly\Sdk\Model\LeadSource::from([])
    ->setSource('TestSource')
    ->setCampaign('TestCampaign');
```

File: openapi/code_samples/PHP-SDK-3.0/password-tokens/post.php [hunk 9]
```diff
@@ -0,0 +1,10 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$resetPasswordTokenForm = Rebilly\Sdk\Model\ResetPasswordToken::from([]);
+$resetPasswordTokenForm->setUsername('username')
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/password-tokens/post.php:4]:
```suggestion
$resetPasswordTokenForm = new Rebilly\Sdk\Model\ResetPasswordToken();
$resetPasswordTokenForm->setUsername('username')
```

File: openapi/code_samples/PHP-SDK-3.0/password-tokens@{id}/get.php [hunk 10]
```diff
@@ -0,0 +1,3 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$passwordToken = $service->customerAuthentication()->getResetPasswordToken('tokenId');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/password-tokens@{id}/get.php:3]:
```suggestion
$passwordToken = $service->customerAuthentication()->getResetPasswordToken('tokenId');

```

File: openapi/code_samples/PHP-SDK-3.0/tags/post.php [hunk 11]
```diff
@@ -0,0 +1,11 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$tagForm = Rebilly\Sdk\Model\Tag::from([]);
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/tags/post.php:3]:
```suggestion
$tagForm = new Rebilly\Sdk\Model\Tag();
```

File: openapi/paths/customers@{id}@lead-source.yaml [hunk 12]
```diff
@@ -98,3 +104,6 @@ delete:
     - lang: JavaScript
       source:
         $ref: '../code_samples/JavaScript/customers@{id}@lead-source/delete.js'
+    - lang: PHP-SDK-3.0
+      source:
+        $ref: '../code_samples/PHP-SDK-3.0/customers@{id}@lead-source/delete.php'
```
Comment (justnero) [inline openapi/paths/customers@{id}@lead-source.yaml:109]:
```suggestion
        $ref: '../code_samples/PHP-SDK-3.0/customers@{id}@lead-source/delete.php'

```

File: openapi/paths/password-tokens.yaml [hunk 13]
```diff
@@ -74,3 +77,6 @@ post:
     - lang: JavaScript
       source:
         $ref: ../code_samples/JavaScript/password-tokens/post.js
+    - lang: PHP-SDK-3.0
+      source:
+        $ref: ../code_samples/PHP-SDK-3.0/password-tokens/post.php
```
Comment (justnero) [inline openapi/paths/password-tokens.yaml:82]:
```suggestion
        $ref: ../code_samples/PHP-SDK-3.0/password-tokens/post.php

```

**Review by justnero — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**